src/proxy/IRingbuffer.ts

```diff
@@ -0,0 +1,124 @@
+import * as Promise from 'bluebird';
+import {DistributedObject} from '../DistributedObject';
+import {OverflowPolicy} from '../core/OverflowPolicy';
+
+export interface IRingbuffer<E> extends DistributedObject {
```

Comment:
I think a language convention. We choose to use I notation for interfaces based on lang.